geonode_mapstore_client/client/js/components/ResourceStatus/ResourceStatus.jsx

```diff
@@ -20,18 +25,26 @@ const ResourceStatus = ({ resource = {} }) => {
         isDeleting,
         isDeleted
     } = getResourceStatuses(resource);
+
+    const getTitle = (status) => {
+        const { isApproved: approved, isPublished: published } = status;
+
+        if (!approved && published) {return <Message msgId="gnhome.pendingApproval" />}
+        if (!approved && !published) {return <Message msgId="gnhome.unApprovedunPublished" />}
+        if (!published && !approved) {return <Message msgId="gnhome.unpublished" />}
+
+        return '';
+    };
+
     return !isEmpty(resource)
         ? (
             <p className="gn-resource-status-text">
-                {(!isProcessing && !isApproved) && <span className="gn-resource-status gn-resource-status-pending" >
-                    <Message msgId="gnviewer.notApproved" />
-                </span>}
-                {(!isProcessing && !isApproved && !isPublished) && <span className="gn-resource-status gn-resource-status-pending" >
-                    {' '}-{' '}
-                </span>}
-                {(!isProcessing && !isPublished) && <span className="gn-resource-status gn-resource-status-pending" >
-                    <Message msgId="gnhome.unpublished" />
-                </span>}
+                {
+                    (!isProcessing && (!isApproved || !isPublished)) && 
+                        <ButtonWithTooltip variant="default" className="gn-resource-status gn-status-button" tooltip={getTitle({ isApproved, isPublished })} style={{ marginRight: (isDeleting || isDeleted || isCopying) && '0.4rem' }} tooltipPosition="top">
```

Comment:
The tooltip is not working. I think the button needs the pointer events set to auto. Anyway now is convered also by the link to the card.

![image](https://user-images.githubusercontent.com/19175505/151008727-bfe142e3-0cd0-4eb9-beff-0a47980fb322.png)